### PR TITLE
Add a few small GUI fixes

### LINKED ISSUE #1417: Fix some minor QT issues
Update the legend
Real selection now possible by mouse and keyboard 
Updated label on tab
Indeterminate now default

### PR DESCRIPTION
**Pull request by frode-aarstad**
Resolves #1417

### CODE REVIEW

File: ert_gui/simulation/view/realization.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def __init__(self, iter: int, parent=None) -> None:
         self._delegateHeight = 50
 
         self._real_view = QListView(self)
-        self._real_view.clicked.connect(lambda e: self.clicked.emit(e))
+        # self._real_view.clicked.connect(lambda e: self.clicked.emit(e))
```
Comment (jondequinor) [inline ert_gui/simulation/view/realization.py:29]: What is the intent here?

Comment (jondequinor) [reply ert_gui/simulation/view/realization.py:29]: Seems you replaced this with `currentChanged`? If so, just remove it.

**Review by jondequinor — APPROVED** [review]
Merge whenever! Looks good!

**Comment by frode-aarstad** [comment]
I have added a label with "Total progress" at the top of the dialog. This was not in the old one. Should we keep or remove ? Thoughts @jondequinor

**Comment by jondequinor** [comment]
I definitely think it improves things.

**Pull request merged**